extensions/sql-bindings/src/services/azureFunctionsService.ts

```diff
@@ -139,19 +139,26 @@ export async function createAzureFunction(connectionString: string, schema: stri
 			});
 
 			// check for the new function file to be created and dispose of the file system watcher
-			const timeoutForFunctionFile = utils.timeoutPromise(constants.timeoutAzureFunctionFileError);
+			const timeoutForFunctionFile = utils.timeoutPromise('');
```

Comment:
Why are you removing the string here?

Reply:
This is so the user does not get any timeout error message when they exit out of the createFunction steps.